src/test/java/io/cdap/plugin/ariba/source/metadata/proto/PropertiesTest.java

```diff
@@ -238,19 +238,25 @@ public void testGetNonObjectFields() {
   }
 
   @Test
-  public void teatValidateFields() throws AribaException, IOException {
+  public void teatValidateFields() {
     MockFailureCollector collector = new MockFailureCollector();
     AribaConnectorConfig connectorConfig = pluginConfig.getConnection();
-    connectorConfig.validateToken(collector);
-    collector.getOrThrowException();
-    Assert.assertEquals(0, collector.getValidationFailures().size());
+    try {
+      connectorConfig.validateToken(collector);
+      Assert.fail("Validation exception will not be thrown if credentials are correct.");
+    } catch (ValidationException e) {
+      Assert.assertEquals(1, collector.getValidationFailures().size());

```

Comment:
Looks like we are only testing validation failure. Can you also add a test for the success case (token validation is successful)?

Reply:
Added the test cases for both the types of failures. Positive test case can't be added as the real call for OkHttpClient can’t be mocked due to which we will always get atleast 1  validation failure.

Reply:
Done